packages/front-end/components/Auth/InAppHelp.tsx

```diff
@@ -106,8 +106,7 @@ export default function InAppHelp() {
                   className="btn btn-premium font-weight-normal my-2 w-100"
                   onClick={() => setUpgradeModal(true)}
                 >
-                  Start {isCloud() ? "Free" : "Enterprise"} Trial{" "}
-                  <GBPremiumBadge />
+                  Start Free Trial <GBPremiumBadge />
```

Comment:
Higher up in this file we set `const showUpgradeModal =
    config && !hasCommercialFeature("livechat") && isCloud();` and we are only showing this section if showUpgradeModal is true, so this check is superfulous.